yarn.lock

```diff
@@ -0,0 +1,70 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please share the reason for removing package-lock.json and adding yarn.lock

Reply:
I switched to yarn.

Reply:
I saw that. I'm asking if there's a reason to do it.
How does it fulfill the goal of removing a dependency that didn't work in your environment?